applications/nrf_desktop/Kconfig.defaults

```diff
@@ -24,6 +24,18 @@ config DESKTOP_LTO_DEFAULTS
 	help
 	  nRF Desktop enables LTO to limit memory usage and improve performance.
 
+config DESKTOP_MCUBOOT_RAM_LOAD_MCUBOOT_RAM_START_ADDR
+	hex
+	depends on MCUBOOT_BOOTLOADER_MODE_RAM_LOAD
+	depends on $(dt_nodelabel_exists,cpuapp_sram_mcuboot_ram_region)
+	default $(dt_nodelabel_reg_addr_hex,cpuapp_sram_mcuboot_ram_region)
```

Comment:
we start to have several devicetree labels trying to achieve similar thing, which is to have the code executed / placed at a specific location.

Until we have some more common way to achieve this on higher level, then I would like to know why the existing `CONFIG_SRAM_OFFSET` is not suitable in this particular case for the MCUboot build ?

@nordicjm please have a look at this, and do some thinking regarding devicetree node used for features like this one, as well as the multicore TCM  etc.
Other places of interest: https://github.com/zephyrproject-rtos/zephyr/pull/91591 github.com/nrfconnect/sdk-nrf/pull/25737

Not blocking comment for this PR, due to closeness to rc, but be prepared for follow-up work in this area.

Reply:
This Kconfig option is only used in the context of the following linker script:

applications/nrf_desktop/linker/mcuboot_ram_load.ld

The CONFIG_SRAM_OFFSET would be correct in the context of the MCUboot image. However, in the MCUboot image context, we do not have access to the `__rom_region_end` symbol of the application image.